elements/rh-disclosure/rh-disclosure.ts

```diff
@@ -1,52 +1,47 @@
 import { LitElement, html, isServer } from 'lit';
 import { customElement } from 'lit/decorators/custom-element.js';
+import { query } from 'lit/decorators/query.js';
+
+import '@rhds/elements/rh-icon/rh-icon.js';
 
 import styles from './rh-disclosure.css';
 
 /**
  * @summary A disclosure is a widget that enables content to be either collapsed (hidden) or expanded (visible).
- * @slot - Place the `details`, `summary`, and `.details-content` elements in the default slot.
+ * @slot - Place the content you want to disclose in the default slot. This content is hidden by default.
+ * @slot summary-label - The title of the disclosure
```

Comment:
if we had a `summary` slot but also needed a `label` for that `summary`, then I would understand the need to hyphenate, but in this instance, maybe we can get away with just `summary`, for brevity